src/lib.rs

```diff
@@ -8,7 +8,11 @@
 //! unified APIs across all platforms. To get started, you simply create an [`Application`] first:
 //!
 //! ```no_run
+//! # use wry::Application;
+//! # fn main() -> Result<(), wry::Error> {
 //! let application = Application::new()?;
+//! # Ok(())
+//! # }
```

Comment:
Why were these added? They seem unrelated to this PR.

Reply:
Tried running `cargo test` in the root, and got failing doctests. Thought I'd fix them while I was at it, but yeah they are unrelated. 

Maybe I should move these changes into a new PR if need be